datadog-setup.php

```diff
@@ -444,6 +444,19 @@ function parse_validate_user_options()
     ];
     $options = getopt($shortOptions, $longOptions);
 
+    global $argc;
```

Comment:
I don't think there is much better at this time we can do. Even in this case, though, we should probably print the help message. WDYT?

Reply:
It should print the help message because of the `print_error_and_exit(..., true)`, I think?

Reply:
I had totally forgotten of that option that I had recently added 😄